nncf/tensor/functions/torch_linalg.py

```diff
@@ -40,6 +40,19 @@ def _(a: torch.Tensor) -> torch.Tensor:
     return torch.linalg.inv(a)
 
 
+@linalg.pinv.register(torch.Tensor)
+def _(a: torch.Tensor) -> torch.Tensor:
+    """
+    Consider using torch.linalg.lstsq() if possible for multiplying a matrix on the left by the pseudo-inverse, as:
```

Comment:
Why note from torch documentation added as docstring?
If you think that this note can be useful, add it as comment

Reply:
Done